src/Controller/Hello.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace App\Controller;
+
+use Symfony\Bundle\FrameworkBundle\Controller\ControllerTrait;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Routing\Annotation\Route;
+
+class Hello
+{
+    use ControllerTrait;
+
+    /**
+     * @Route("/", name="homepage")
+     */
+    public function __invoke(Request $request): Response
```

Comment:
I'm not sure we should encourage the "single-action-controller" principle, from a DX point of view this is terrible!